src/main/java/com/shaneschulte/mc/clanarena/arena/Region.java

```diff
@@ -0,0 +1,37 @@
+package com.shaneschulte.mc.clanarena.arena;
+
+import org.bukkit.Location;
+
+public class Region {
+
+    private Location pos1, pos2;
+
+    public Region(Location pos1, Location pos2) {
+        this.pos1 = pos1;
+        this.pos2 = pos2;
+    }
+
+    public boolean inArea(Location targetLocation) {
+        return inArea(targetLocation, true);
+    }
+
+    public boolean inArea(Location targetLocation, boolean checkY) {
```

Comment:
a less dangerous way to do this is to implement a helper function that checks for overlap in a single dimension; then you can simply && those checks together in this function